framework/test/gtest/BundleHooksTest.cpp

```diff
@@ -178,14 +178,19 @@ TEST_F(BundleHooksTest, TestEventHook)
   bundleA.Start();
 
   //Test for received load bundle events
-  ASSERT_TRUE(bundleListener.events.size() == 4 ||
-              bundleListener.events.size() == 3);
+#ifdef US_BUILD_SHARED_LIBS
+  ASSERT_EQ(bundleListener.events.size(), 4);
+#else
+  ASSERT_GE(static_cast<int>(bundleListener.events.size()), 3);
+#endif
```

Comment:
Looks like it can be generalized to greater than or equal to 3. Or it's important make sure US_BUILD_SHARED_LIBS must be equal 4?

Just wondering why does static_cast needed ASSERT_GE.

Reply:
Its important to ensure the exact number of events were received.